packages/vue/src/types.ts

```diff
@@ -46,6 +46,14 @@ export interface Options extends TracingOptions, BrowserOptions {
 
   /** {@link TracingOptions} */
   tracingOptions?: Partial<TracingOptions>;
+
+  /**
+   * What to use for route labels.
+   * By default, we use route.name (if set) and else the path.
+   *
+   * Default: 'name'
+   */
+  routeLabel: 'name' | 'path';
```

Comment:
h: Instead of making this a top level option, wdyt about adding this as an option to the routing instrumentation?